framework/utils.py

```diff
@@ -32,6 +32,32 @@ def is_internal_ip(ip):
     parsed_ip = IPAddress(ip)
     return parsed_ip.is_private
 
+def get_rank(user_rank, owtf_rank):
```

Comment:
Apparently no, there is no other reference where the ranking scale is defined (except hardcoded in https://github.com/owtf/owtf/blob/master/framework/interface/templates/plugin_report.html)